phpBB/includes/acp/acp_bbcodes.php

```diff
@@ -269,6 +269,7 @@ function main($id, $mode)
 
```

Comment:
Ah!
phpbb/textformatter/s9e/factory.php belongs to phpBB itself... Ok, then it makes sense.